bin/internal/shared.bat

```diff
@@ -145,6 +147,7 @@ GOTO :after_subroutine
         ECHO        Either pwsh.exe or PowerShell.exe must be in your PATH. 1>&2
         EXIT 1
     )
+    SET /A retries+=1
```

Comment:
Rename this to `dart_sdk_retries` to avoid potential name conflicts with other retry counts.